core/mixins/_retina-bg-image.scss

```diff
@@ -17,11 +17,13 @@
  */
 
 
-@mixin retina-bg-image($image, $width, $height) {
-  @media (-o-min-device-pixel-ratio: 5/4),
-         (-webkit-min-device-pixel-ratio: 1.25),
-         (min-resolution: 120dpi) {
-           background-image: url($image);
-           background-size: $width $height;
+@mixin retina-bg-image($image, $width: auto, $height: auto, $resolution: $retina-resolution) {
+  $width: if($width, $width, auto);
+  $height: if($height, $height, auto);
+  $resolution: if(unitless($resolution), $resolution * 96dpi, $resolution);
+  
+  @media (min-resolution: $resolution) {
+    background-image: url($image);
+    background-size: $width $height;
   }
```

Comment:
> What do I pass in if I want to use the Scally resolution default

You don't have to pass in anything. As it's the last argument and already has the default `$retina-resolution` set as `$resolution` it's effectively optional.